docs/api-reference/next.config.js/resolve-alias.md

```diff
@@ -0,0 +1,28 @@
+---
+description: Configure Next.js with Turbopack to alias module resolution
+---
+
+# Resolve Alias (`next --turbo` only)
+
+Through `next.config.js`, Turbopack can be configured to modify module resolution through aliases, similar to Webpack's [`resolve.alias`](https://Webpack.js.org/configuration/resolve/#resolvealias) configuration.
```

Comment:
```suggestion
Through `next.config.js`, Turbopack can be configured to modify module resolution through aliases, similar to webpack's [`resolve.alias`](https://webpack.js.org/configuration/resolve/#resolvealias) configuration.
```

webpack is always lowercase

Reply:
This was my preference too. On https://turbo.build we use uppercase consistently. I'll change it for the Next.js site and we'll revisit on the Turbo side.